GotChat/Controller/MessageViewController.swift

```diff
@@ -1,216 +0,0 @@
-//
-//  ViewController.swift
-//  GotChat
-//
-//  Created by Ilshat Khairakhun on 11/11/20.
-//
-
-import UIKit
-import Firebase
-import EasyPeasy
-
-class MessageViewController: UITableViewController {
-    
-    let loginVC = LoginViewController()
```

Comment:
don't use two view controller inside one if you don't  present them together in same view